synapse/module_api/__init__.py

```diff
@@ -1230,6 +1231,42 @@ def looping_background_call(
                 f,
             )
 
+    def delayed_background_call(
+        self,
+        msec: float,
+        f: Callable,
```

Comment:
`f` and `msec` are swapped from `looping_background_call` -- that might be confusing. should we swap them?

Reply:
on the other hand it's currently the same order as `call_later` and I have a slight preference for keeping the function directly adjacent to the arguments that get passed through (otherwise as a reader in this sort of situation I find it confusing which arguments are passthroughs and which are arguments to `delayed_background_call`..) . Ultimately I don't mind much, so the choice is yours?

Reply:
I don't have a strong opinion. Keeping it the same as the internals is nice for us, but keeping it the same as `looping_background_call` is probably nicer for our users? 🤷 Anyway, not a big deal in my mind.